---
title: IBMQBackendApiError
description: API reference for qiskit.providers.ibmq.IBMQBackendApiError
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.IBMQBackendApiError
---

<span id="qiskit-providers-ibmq-ibmqbackendapierror" />

# qiskit.providers.ibmq.IBMQBackendApiError

<Class id="qiskit.providers.ibmq.IBMQBackendApiError" isDedicatedPage={true} github="https://github.com/qiskit/qiskit-ibmq-provider/tree/stable/0.12/qiskit/providers/ibmq/exceptions.py" signature="IBMQBackendApiError(*message)" modifiers="exception">
  Errors that occur unexpectedly when querying the server.

  Set the error message.

  ### with\_traceback

  <Function id="qiskit.providers.ibmq.IBMQBackendApiError.with_traceback" signature="with_traceback()">
    Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.
  </Function>
</Class>

